ci/steps.lib.yml

```diff
@@ -105,6 +105,7 @@ config:
     INCLUDE_SLOW_TESTS: #@ str(not quick).lower()
     RUN_NIGHTLY_TESTS: #@ str(test_nightly).lower()
     BUILD_VERBOSE: "1"
+    VERBOSE: "1"
```

Comment:
Any reason to make this an environment variable instead of a command-line option?

Reply:
Yeah, a flag would be preferable for ease of typing and for discoverability.  I just couldn't figure out how to make it work: the interactions between `unit/main.c`, `config.h`, `unit/config_parse_test.c`, and `unit/splinter_test.c` were just too messy for me to untangle in the couple hours that I spent on this.  I'd welcome some help on that from @gapisback or anyone else.

Reply:
Yes, we'll return to this in future work